### PR TITLE
Remove `default` case

### PR DESCRIPTION
**Pull request by tnasu**
## Description

Remove `default` case since shouldn't abandon message although blocking sending a message into golang channel

Closes: #316

### CODE REVIEW

**Review by kokeshiM0chi — APPROVED** [review]
LGTM

File: p2p/peer.go [hunk 1]
```diff
@@ -399,19 +399,23 @@ func createMConnection(
...
+				// Remove `default` case since shouldn't abandon message although blocking sending message into golang channel
+				//default:
+				//	// if the channel is full, we abandon this message
+				//	// Should check `config.Config.XxxBufSize`
+				//	p.Logger.Error("Lost the message since BaseReactor.recvMsgBuf is full",
+				//		"reactor", reactor,
+				//		"msgBytes.len", len(msgBytes), "msgBytes", fmt.Sprintf("%X", msgBytes))
+				//	p.metrics.NumAbandonedPeerMsgs.With(labels...).Add(1)
+				}
+			*/
```
Comment (torao) [inline p2p/peer.go:418]: I think these can be removed, but is there any reason to leave it commented out?

**Review by torao — DISMISSED** [review]

File: p2p/peer.go (hunk 1 above)
Comment (tnasu) [reply p2p/peer.go:418]: I want that nobody uses `select case and default`.

Comment (torao) [reply p2p/peer.go]: I thought it'd be better to leave only a one-line comment such that "fixed to block and wait to avoid abandoned messages".

**Review by torao — APPROVED** [review]
LGTM

**Review by kokeshiM0chi — APPROVED** [review]

**Pull request merged**